src/interfaces/index.ts

```diff
@@ -86,3 +90,136 @@ export type DocumentContextWithSession = DocumentContext & {
 };
 
 export type IncomingMessageWithSession = IncomingMessage & Session;
+
+export interface SessionContents {
+    body: any;
+}
+
+export interface SalesOfferPackage {
+    name: string;
+    description: string;
+    purchaseLocations: string[];
+    paymentMethods: string[];
+    ticketFormats: string[];
+}
+
+export interface Product {
+    productName: string;
+    productPrice: string;
+    productDuration?: string;
+    productValidity?: string;
+}
+
+export interface PeriodProductData {
+    products: ProductDetails[];
+}
+
+export interface FlatFareProductData {
+    products: FlatFareProductDetails[];
+}
+
+export interface ProductData {
+    products: Product[];
+}
+
+export interface PeriodExpiryWithErrors {
+    errorMessage: string;
+}
```

Comment:
Same here as with `SelectSalesOfferPackageWithError`

Reply:
I have moved the interface to periodValidity.ts. I haven't change it to comprise a ErrorInfo[] under a key of errors as we only have a single error message on periodValidity.ts.